nodebox/graphics/context.py

```diff
@@ -2535,7 +2544,8 @@ def __init__(self):
 
     def _deepcopy(self, value):
         if isinstance(value, FunctionType):
-            return instancemethod(value, self)
+            return getattr(self, value.__name__) # Replaces instancemethod(value, self)
```

Comment:
This might be why `draw()` doesn't work. I noticed that `value` is usually (almost always?)  the `draw` function? Might be related to why `draw()` doesn't actually draw anything, because i return it in the wrong way.